src/executor.ts

```diff
@@ -49,21 +49,21 @@ process.on('exit', () => {
   }
 });
 
-const spawnServer = (cwd, serverPath, debugOpt, logger) => {
-  server = spawn('node', [debugOpt, serverPath], { stdio: [0, 1, 2], cwd });
-  logger(`Spawning ${['node', debugOpt, serverPath].join(' ')}`);
+const spawnServer = (cwd, serverPath, options, logger) => {
```

Comment:
Maybe better pass `args` array here instead of two options `serverPath` and `options` and make that array on a higher level. Having `options.debugOpts` as empty string looks imperfect and will have imperfect output in logs.